src/ch/jamiete/hilda/HildaDB.java

```diff
@@ -0,0 +1,63 @@
+package ch.jamiete.hilda;
+
+import java.sql.*;
+import java.util.Properties;
+import java.util.logging.Level;
+
+public class HildaDB {
+    private static final String DATABASE_DRIVER = "com.mysql.jdbc.Driver";
+    private static final String DATABASE_URL = "";
+    private static final String USERNAME = "";
+    private static final String PASSWORD = "";
+    private static final String MAX_POOL = "250";
+
+    private Connection con;
+    private Properties properties;
```

Comment:
Can this not be stored only in the strings and then used with the JDBC methods instead of making a properties file?

Reply:
No.

Reply:
https://docs.oracle.com/javase/8/docs/api/java/sql/DriverManager.html#getConnection-java.lang.String-java.lang.String-java.lang.String-